src/bin/sol-fbp-generator/main.c

```diff
@@ -1888,7 +1888,13 @@ print_usage(const char *program)
         "        Multiple -j can be passed.\n"
         "    -s  Define a function named SYMBOL that will return the type from FBP\n"
         "        and don't generate any main function or entry point.\n"
-        "    -I  Define search path for FBP files\n"
+        "    -I  Define search path for FBP files\n\n"
```

Comment:
Minor trick: If you put the extra "\n" in its own line, the patch wouldn't touch unrelated text for -I. Not very important though.